proxy_test.go

```diff
@@ -200,7 +201,7 @@ func TestServeHTTP_allowed(t *testing.T) {
 	}
 	messenger := NewMockMessenger()
 	proxy := NewProxy(s3conf, NewAlwaysAllow(), messenger, new(tls.Config))
-	//proxy := NewProxy(s3conf, NewValidateFromFile("./dev_utils/users.csv"), NewMockMessenger(), s, new(tls.Config))
+	// proxy := NewProxy(s3conf, NewValidateFromFile("./dev_utils/users.csv"), NewMockMessenger(), s, new(tls.Config))
```

Comment:
```suggestion
```
Better to delete rows than comment them out.

Reply:
Yes you are right in that. This line was there and I just added the space that the linter was requesting. Will remove it